python2/httplib2/__init__.py

```diff
@@ -74,6 +74,8 @@
     ssl_SSLError = getattr(ssl, "SSLError", None)
     ssl_CertificateError = getattr(ssl, "CertificateError", None)
 
+# socks.PROXY_TYPE_SOCKS4, socks.PROXY_TYPE_SOCKS5, socks.PROXY_TYPE_HTTP
+proxy_schemes = { 'socks4' : 1, 'socks5' : 2, 'http' : 3 }
```

Comment:
Please use what `black --diff python2/ python3/` says here

Reply:
Please put this after socks import or somewhere not between two ssl related blocks.